src/index.js

```diff
@@ -1,15 +1,20 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
-import { BrowserRouter } from 'react-router-dom';
+import { BrowserRouter, Route, Routes } from 'react-router-dom';
 
 import App from './App';
 
```

Comment:
This implementation of react-router is breaking the flow of the project, the routing should be done in the App.js file instead, the header is supposed to show on every page but you are rendering the header only on the home page

Reply:
But I do not have the other pages. Because the issue was only for the Profile page alone.